TRB_label_assist/SVM_decision_boundaries.ipynb

```diff
@@ -270,9 +269,9 @@
     ],
     "metadata": {
         "kernelspec": {
-            "display_name": "emission-private-eval",
```

Comment:
why did these names change? Are you not running this notebook in the correct environment?

Reply:
the environments is correct.However, it shows up as `emission-private-eval` ( and not Python3) on the json of the notebook as soon as I run even 1 cell. This might have slipped passed me in one of the previous commits as I usually reset this before committing.

Reply:
And now it is `Python3` instead of `Python 3`. Argggh!!

@humbleOldSage **Can you please revert these as well?**

We should then do a separate one-time change to make the metadata current, with **no other changes**, similar to 
https://github.com/e-mission/e-mission-phone/commit/bb73676c7245b68b9a8784204b68d71ba0af5267

Reply:
In general, when you respond like this, please clarify what (if anything) you have done to address the comment.
I can then verify it and resolve.

Reply:
This has still not been resolved

Reply:
Reverted this back to Python 3 using git file history.